packages/web/package.json

```diff
@@ -0,0 +1,43 @@
+{
+    "name": "@concordium/web",
```

Comment:
Yes... do you like `web3`? I personally think it doesn't say anything about the scope of the package, but if anyone has a strong opinion I don't care. I think with the `sdk` postfix, we align with our other releases instead of aligning with the general blockchain community, so that's the stuff we have to weigh.